servers/phantomhq/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "phantomhq",
+  "name": "PhantomHQ",
+  "description": "A brand new journey...",
+  "addresses": [
+    "play.phantomhq.club",
```

Comment:
You don't need the subdomain Lunar Client auto detects that! Please just have "phantomhq.club" in the addresses array.